tts-deploy-on-embedded.ipynb

```diff
@@ -0,0 +1,280 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "caa47ff5",
+   "metadata": {},
+   "source": [
+    "# TTS Deploy on embedded\n",
+    "\n",
+    "In this tutorial we will explain the process of generating TTS RMIR from acoustic_model.riva and vocoder.riva files for embedded machines. RMIR (Riva Model Intermediate Representation) is an intermediate file that has all the necessary artifacts (models, files, configurations, and user settings) required to deploy a Riva service.  \n",
+    "\n",
+    "In this tutorial we will use pretrained [Fastpitch.riva](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/tao/models/speechsynthesis_en_us_fastpitch_ipa) and [HifiGan.riva](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/tao/models/speechsynthesis_en_us_hifigan_ipa). These can be replaced with any custom acoutic_model or vocoder riva files. [`nemo2riva`](https://docs.nvidia.com/deeplearning/riva/user-guide/docs/model-overview.html#export-models-with-nemo2riva) can be used to generate .riva files from nemo checkpoints.  \n",
+    "\n",
+    "The RMIR generated in this tutorial can be deployed using [riva_quickstart](https://docs.nvidia.com/deeplearning/riva/user-guide/docs/quick-start-guide.html)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "14e5a5e4",
+   "metadata": {},
+   "source": [
+    "import pathlib to get the paths"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "34284f33",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pathlib"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "670aea87",
+   "metadata": {},
+   "source": [
+    "### Download models\n",
+    "We will download pretrained [Fastpitch](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/tao/models/speechsynthesis_en_us_fastpitch_ipa) and [HifiGan](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/tao/models/speechsynthesis_en_us_hifigan_ipa) models from ngc. You can replace these models with the paths of your custom model, incase of custom models."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "eca2a574",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!ngc registry model download-version \"nvidia/tao/speechsynthesis_en_us_fastpitch_ipa:deployable_v1.0\"\n",
+    "!ngc registry model download-version \"nvidia/tao/speechsynthesis_en_us_hifigan_ipa:deployable_v1.0\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d93f949e",
+   "metadata": {},
+   "source": [
+    "Download the riva model repository files from ngc."
```

Comment:
Can you explain what this repository is for rather than calling it a `riva model repository files`?